### ISSUE TITLE
Containers cannot access internet

### ISSUE THREAD

**Issue by ekeryelleven** [docker/docker#12372]
When I run containers, they cannot access internet (`apt-get update` timing out, can't get the python web app from the user-guide to work, ...)

I'm running CentOS 6.6.

I'm behind a corporate proxy. I've added HTTP_PROXY and HTTPS_PROXY in the /etc/sys/config file, and the docker daemon is now able to reach the Hub without problem.

I'm running Docker version 1.4.1

The base image I'm using is ubuntu:14.10

The command I'm using is
`sudo docker run -t -i ubuntu:14.10 sudo apt-get update`, and it keeps stuck on
`0% [Connecting to archive.ubuntu.com (91.189.91.23)]`.

ipv4 forwarding is enabled on both the host and the container.

ufw isn't installed neither on the container nor on the host.

I'm not competent at all when it comes to networking.

I've been through those links and it still doesn't work (though I might have missed something since I don't know much about networking and all).
https://github.com/docker/docker/issues/866
http://odino.org/cannot-connect-to-the-internet-from-your-docker-containers/
http://askubuntu.com/questions/302050/nat-not-working-in-a-container

I'd be glad to provide any output (/etc/resolve.conf, ifconfig, ...) if someone is willing to help.
Thanks for your attention!

**Comment by thaJeztah** [comment]
I think this has been resolved in docker 1.5, see https://github.com/docker/docker/pull/9951 (and some issues linking to that)

Are you able to test using the docker 1.5 release, or the current 1.6 release candidate (https://github.com/docker/docker/pull/11635#issuecomment-92482581) to see if that resolves this issue for you?

**Comment by ekeryelleven** [comment]
Sure, I'll do that tomorrow. Thanks for the quick answer :)

**Comment by ekeryelleven** [comment]
Ok I've solved the problem, and it wasn't at all related to docker.

I've uninstalled my previous version of Docker (1.4.1), and installed version 1.5.0
I've then launched a ubuntu container. `apt-get update` still wasn't working, but by setting the right proxy for it I got it to work. I did the same in a centOS container to get yum working through the proxy, and everything went fine.

Really sorry for this useless "issue" !
